src/tests/ie_test_utils/common_test_utils/data_utils.cpp

```diff
@@ -202,7 +206,132 @@ size_t byte_size(const InferenceEngine::TensorDesc &tdesc) {
     auto dims = tdesc.getDims();
     return prc.size() * std::accumulate(std::begin(dims), std::end(dims), (size_t)1, std::multiplies<size_t>());
 }
+OPENVINO_SUPPRESS_DEPRECATED_END
+
+
+void fill_data_with_broadcast(ov::Tensor& tensor, ov::Tensor& values) {
+    constexpr size_t MAX_N_DIMS = 7;  // Suppose it's enough
```

Comment:
duplication as for old one. Could we remove old one?

Reply:
Old one is used in StaticConditionLayerGPUTest and LoopTest (cpu func), so we have to keep it now.

can I split data_utils.cpp to new and legacy parts to hide old methods?